src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null) {
+            if (userService.findByEmail(email).getPassword().equals(password)) {
+                return true;
+            } else {
+                return false;
+            }
+        }
         return false;
```

Comment:
try to rewrite this without using true/false literals, only ternary operator

Reply:
Fixed. Applied there ternary operator, but Intellij IDEA suggested to simplify the expression even more, applied suggestion.